{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/js/utils.js

```diff
@@ -66,17 +66,6 @@ export function externalLinks () {
 }
 
 /*
-  Media breakpoints
- */
-export const mediaBreakpoints = {
-  'sm': 600,
-  'md': 900,
-  'lg': 1200,
-  'xlg': 1800
-}
-
-
-/*
```

Comment:
Seems it is, never thought it had been used. Will add it back, i removed as i want to see if i can programatically create these instead of manually. Will do that in another PR>